oibackoff/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for oibackoff
+// Project: https://github.com/chilts/oibackoff
+// Definitions by: Joshua DeVinney <https://github.com/geoffreak>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export declare function backoff(opts: IBackoffOptions): IBackoffInstance;
+
+export declare interface IBackoffOptions {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html#naming-conventions
> In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.